sdk/security/keyvault/azsecrets/options.go

```diff
@@ -27,18 +27,21 @@ type GetSecretOptions struct {
 // ListDeletedSecretPropertiesOptions contains the optional parameters for the Client.NewListDeletedSecretPropertiesPager
 // method.
 type ListDeletedSecretPropertiesOptions struct {
-	// placeholder for future optional parameters
+	// Maximum number of results to return in a page. If not specified the service will return up to 25 results.
+	MaxResults *int32
```

Comment:
Can you please do this in a separate PR? It's related to fakes only by our process

Reply:
I wish, but I don't think that's logistically possible. The typespec needs to be correct for me to generate fakes. So I need to either repress max results in the typespec or keep it.